### DISCUSSION TITLE
Example of plot being updated by a (long-)running computation?

### DISCUSSION THREAD

**Discussion by robertfeldt** [fonsp/Pluto.jl#1166] [Questions about using Pluto]
I wonder if anyone has seen some example Pluto notebooks where a plot is being updated as a long-running computation is going on? I've been playing with using Plots.jl and plotly but it's not trivial getting state changes to propagate nicely between cells and the re-plotting to not be very resource-hungry. Inspirational examples appreciated! :)

Ideally, I'm thinking of building a kind of control-interface for starting, stopping and pausing a long-running computation while plotting its progress in a few different plots/graphs.

**Comment by fonsp** [comment]
There is a small example here that might be useful: https://gist.github.com/fonsp/971666be05d34bfc1254d69756ac3b1f

**Reply by robertfeldt** [reply]
Thanks, this is useful. So far in my experiments I ended up with using a Clock to drive the redraws (if there has been any updates that need to be redrawn).

**Reply by robertfeldt** [reply]
Hmm, my clock-based idea doesn't work though since the long-running computation runs to completion before the plot is updated anyway. Ok, so I'll check other question on asynchronous updates then, I guess...
